File: swarm_input/hello2.rb
class FormalGreeter
  TITLES = { en: "Mr.", de: "Herr", fr: "M." }
  def initialize(lang = :en); @lang = lang; end
  def greet(name)
    title = TITLES.fetch(@lang, "")
    puts "Hello, #{title} #{name}!"
  end
end

[:en, :de, :fr].each do |lang|
  FormalGreeter.new(lang).greet("Smith")
end

File: swarm_input/hello2.py
class FormalGreeter:
    TITLES = {"en": "Mr.", "de": "Herr", "fr": "M."}

    def __init__(self, lang="en"):
        self.lang = lang

    def greet(self, name):
        title = self.TITLES.get(self.lang, "")
        print(f"Hello, {title} {name}!")

for lang in ["en", "de", "fr"]:
    FormalGreeter(lang).greet("Smith")
